lib/agent.js

```diff
@@ -261,6 +262,17 @@ class Agent {
             agentVersion: this.config.version,
             licensed: this.config.licensed
         }
+        if (this.launcher?.readPackage) {
+            const { modules } = this.launcher.readPackage()
+            if (module['node-red'] !== 'latest') {
```

Comment:
```suggestion
            if (modules['node-red'] !== 'latest') {
```

module is the actual `module` -  i suspect this should be `modules`